public/static/locales/fr/cognito-errors.json

```diff
@@ -23,7 +23,7 @@
     "linkText": "Si vous n’avez pas de compte, créez-en un maintenant."
   },
   "2FALockedOutSession": {
```

Comment:
I think you forgot to add a new key for `2FASessionExpired` in the French translation file